microsoft-edge/devtools-guide-chromium/about-tools.md

```diff
@@ -49,7 +49,6 @@ Microsoft Edge DevTools includes the following tools.
 | **Inspect** tool | Use the **Inspect** tool to see information about an item within a rendered webpage.  When the **Inspect** tool is active, you _hover_ over items in the webpage, and DevTools adds an information overlay information and grid highlighting on the webpage. | [Analyze pages using the Inspect tool](css/inspect.md) |
 | **Issues** tool | The **Issues** tool automatically analyzes the current webpage, reports issues grouped by type, and provides documentation to help explain and resolve the issues. | [Find and fix problems using the Issues tool](issues/index.md) |
 | **JavaScript Profiler** tool | This tool has been replaced by the **Performance** tool and the **Memory** tool. | [JavaScript Profiler tool](javascript-profiler/javascript-profiler-tool.md) |
-| **Layers** tool | The **Layers** tool represents the webpage in a 3D space.  The **Layers** tool has essentially been replaced by the **3D View** tool. | [Layers tool](layers/layers-tool.md) |
```

Comment:
The redirects file to update: edge-developer\.openpublishing.redirection.json

Reply:
I added a redirection rule to the json file and tested it on the preview site. It seems to work fine.